docs/install/index.md

```diff
@@ -2,43 +2,57 @@
 
 Indicate your preferred configuration. Then, follow the customized commands to install *MXNet*.
 
+  <div class="dropdown">
+    <button class="btn current-version btn-primary dropdown-toggle" type="button" data-toggle="dropdown">v1.2.0
+    <span class="caret"></span></button>
+    <ul class="dropdown-menu opt-group">
+      <li class="opt active versions"><a href="#">v1.2.0</a></li>
```

Comment:
auto-generated? @aaronmarkham

Reply:
no

Reply:
Sorry, I'm indicating that it should be autogenerated :)

Reply:
We debated about that, and it is something we can add, however I'm also trying to reduce complexity of the site build rather than add to it. So, we specifically decided to hardcode it at this time. Then when there's a release we'll have to update this page along with the home page when we do the release announcement.

Reply:
Okay. Please change the release process documentation accordingly